engine/include/cubos/engine/audio/plugin.hpp

```diff
@@ -0,0 +1,40 @@
+/// @dir
+/// @brief @ref audio-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup audio-plugin
+
+#pragma once
+
+#include <cubos/engine/audio/audio.hpp>
+#include <cubos/engine/audio/listener.hpp>
+#include <cubos/engine/audio/source.hpp>
+#include <cubos/engine/prelude.hpp>
+
+namespace cubos::engine
+{
+    /// @defgroup audio-plugin Audio
+    /// @ingroup engine
+    /// @brief Adds audio management to @b Cubos
+    ///
+    /// ## Settings
+    /// - `audio.listenerCount` - number of listeners per audio device (default: `4`).
+    ///
+    /// ## Dependencies
+    /// - @ref settings-plugin
+
+    /// @brief Initializes the audio manager (after @ref settingsTag).
+    CUBOS_ENGINE_API extern Tag audioInitTag;
+
+    /// @brief Systems which add bridges to the audio manager should be tagged with this.
+    CUBOS_ENGINE_API extern Tag audioBridgeTag;
```

Comment:
You don't need this, you should just tag the systems which add bridges with `assetBridgeTag`